fix: plugin config sync from new-joined node when try starting it (#13721)
emqx/emqx: changes/ce/fix-13721.en.md

> Fixed an issue where the plugin configuration file could not be retrieved from other nodes in the cluster when trying to start the plugin and the plugin configuration file did not exist.

Reviewer: ```suggestion
Resolved a retrieval issue with the plugin configuration file from cluster nodes when initiating the plugin without an existing configuration file.
```
